util/logtools/arg_parser.ts

```diff
@@ -0,0 +1,88 @@
+import argparse from 'argparse';
+
+type argsObject = {
```

Comment:
(a pile of tiny nits, but types should be capitalized.  And maybe I'd bikeshed and call this `TimelineArgs`? And also export this?)